tests/python/relay/test_pass_fake_quantization_to_integer.py

```diff
@@ -318,17 +318,26 @@ def test_fake_quantize_global_avg_pool():
 
 
 class TestUnaryQNNOp:
-    def helper_test_fake_quantize_unary_op(self, fp32_op, scale=0.125):
-        x = relay.var("x", shape=[1, 3, 3, 3], dtype="int8")
-        mid_point = relay.const(-128)
+    def helper_test_fake_quantize_unary_op(self, fp32_op):
+        for dtype in ["int8", "uint8"]:
+            x = relay.var("x", shape=[1, 3, 3, 3], dtype=dtype)
 
-        x = relay.qnn.op.dequantize(x, relay.const(scale), mid_point)
-        op = fp32_op(x)
-        op = relay.qnn.op.quantize(op, relay.const(scale), mid_point)
+            zero = -128 if dtype == "int8" else 0
 
-        x_np = np.random.randint(-128, 127, size=[1, 3, 3, 3], dtype="int8")
+            mid_point = relay.const(np.random.randint(0, 255) + zero)
```

Comment:
Can you rename these input_mid_point, input_scale, outptu_scale, etc. for readability?